controller/pkg/remoteenforcer/remoteenforcer_linux.go

```diff
@@ -114,7 +115,7 @@ func (s *RemoteEnforcer) setupEnforcer(req rpcwrapper.Request) (err error) {
 		payload.ExternalIPCacheTimeout,
 		payload.PacketLogs,
 	); err != nil || s.enforcer == nil {
-		return errors.New("unable to setup enforcer: we don't know as this function does not return an error")
+		return fmt.Errorf("Error while initializing remote enforcer, %s", err)
```

Comment:
The error here is actually meaningful and can be consumed, so we should return it